test/test_serde.py

```diff
@@ -556,17 +556,17 @@ def test_pointer_tensor_detail(id):
 
 
 def test_numpy_tensor_serde():
-    syft.serde._serialize_tensor = syft.serde.numpy_tensor_serializer
-    syft.serde._deserialize_tensor = syft.serde.numpy_tensor_deserializer
+    syft.torch_serde._serialize_tensor = syft.torch_serde.numpy_tensor_serializer
```

Comment:
can we change this back. I don't want to know that three files exist for these unless I have to. In syft.serde.__init__.py if you add `from torch_serde import *` it should fix this

Reply:
done it by adding to serde/init.py
```from syft.serde.serde import *
from syft.serde.torch_serde import *
```